setup.py

```diff
@@ -46,8 +46,9 @@ def get_version():
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
     install_requires=[
-        'django>=1.11,<3',
+        'django>=1.11',
         'requests_oauthlib',
+        'sentry_sdk'
```

Comment:
Uhm, this was left out on purpose. Some project may have the old sentry package and installing this would break them. The client supports both the incarnations of sentry

Reply:
hmm.. OK but our app doesn't use sentry at the moment.

Shall I add in the README that sentry needs to be installed for this to work?

Reply:
I see your point and I'm not happy with the choice either, it was based on the fact that django-restframework doesn't list Django as requirement!
Probably the best way of doing this would either be taking out sentry support, which should be in the app, or make it optional

Reply:
Oh yes I see sentry is used as a fallback in the views if raven isn't available.

But raven is listed as a dependency in `setup.py` so I am not sure why we would need sentry

Reply:
I have added @richtier to this PR as I think he added sentry support

Reply:
removed `sentry_sdk`

Reply:
@richtier any thoughts on removing sentry?

Reply:
our app will actually use sentry so maybe keep as is?

Reply:
I was under the impression that is a requirement to use sentry for our apps